src/hooks/useEditorView.ts

```diff
@@ -1,18 +1,37 @@
+import { useForceUpdate } from "./useForceUpdate.js";
```

Comment:
@smoores-dev any idea here why github actions check wouldn't be aligned with my local eslint/vscode lint configuration?  I'm not seeing this error on my end. I'm trying to poke around with whether or not I'm using some global eslint configuration instead of the local one, but I can't see why that would be the case.

Reply:
Hmmmm. There shouldn't be a `node_modules/eslint` in this project at all; it's using plug n play, which doesn't install packages in node modules. That's why there's a `.vscode/settings.json` that configures `"eslint.nodePath": ".yarn/sdks"`; so that the VS Code ESLint extension will use the PnP loader to load the ESLint package.

Reply:
To run the checks locally, you should be able to run `yarn check`

Reply:
But it should also be working with VS Code; it does for me!

Reply:
hmm, I ran `yarn` and I ended up going a node_nodules folder in my directory, looks like I somehow got opted out of PnP for some reason

Reply:
Does the `node_modules` directory actually have packages in it, or just caches like `.cache` and `.vite`? The only way I could imagine yarn not using PnP is if you're using a very old version of yarn. What do you see when you run `yarn --version`?

Reply:
I re-installed everything and `yarn check` gave me a whole host of import order issues for some reason 
```
<img width="1298" alt="image" src="https://github.com/nytimes/react-prosemirror/assets/19535560/4c07f871-09a3-47b8-ae31-437cae193e7a">
```

Reply:
> . What do you see when you run `yarn --version`?

yarn --version gives me `3.4.1`

Reply:
<img width="1051" alt="image" src="https://github.com/nytimes/react-prosemirror/assets/19535560/2859919f-8581-48bd-a1b2-6418c7600f90">

Reply:
Yeah, I'm not sure what's going on here. Has your yarn.lock changed somehow? Did you install newer versions of eslint or the import plugin?

Reply:
I even ran vscode in a devcontainer and eslint gives me the same result.

Reply:
So I just re-cloned the repo. Here's the steps I took (I didn't even open vscode)
- nvm use v18
- yarn 
- yarn check

I get a huge output of import errors, FWIW I don't get the same issue in a .devcontainer version, I just am not sure why they would be different

Reply:
Oh... interesting! Do you have a `~/.yarnrc.yml` that has `nodeLinker: node-modules`? That's the only reason I could imagine it not defaulting to pnp

Reply:
(I see the other thread now, glad we figured it out!)